pom.xml

```diff
@@ -202,6 +202,7 @@
               <exclude>src/site/resources/download_lang.cgi</exclude>
               <exclude>src/site/resources/release-notes/RELEASE-NOTES-*.txt</exclude>
               <exclude>src/test/resources/lang-708-input.txt</exclude>
+              <exclude>nb-configuration.xml</exclude>
```

Comment:
This is not necessary, since RAT has a implicit exclusion list populated from the `.gitignore` file.

Reply:
that file is generated by NetBeans when using it. So no, does not have an AL header.
I've added the file to .gitignore but checkstyle did still complain.

Reply:
Did Checkstyle or Apache RAT fail? Apache RAT should be OK without this line, Checkstyle does not read the configuration of Apache RAT AFAIK.

Reply:
I'll try again without

Reply:
yup, you're right it seems. Works now.